### ISSUE TITLE
Get company logo

### ISSUE THREAD

**Issue by calvinomiguel** [xmyoot/linkedin-scraper#2]
It'd be great if the scraper could get the link of the company's logo too.

**Comment by xmyoot** [comment]
Yes I agree :( unfortunately the logo link returns a 403 forbidden access, there may be work around for this but I'm not sure. Thanks for your interest in the project!
